tests/testfiles/product/zh-CN/browser/chrome/browser/main.ftl

```diff
@@ -0,0 +1,15 @@
+// Variables:
+//   $num - default value of the `dom.ipc.processCount` pref.
+default-content-process-count
+    .label = { $num } (default)
+
+// Plural form
+timeDiffHoursAgo = { $number ->
+        [one] 1 hour ago
+       *[other] { $number } hours ago
+}
```

Comment:
@stasm
Do you have any idea why this plural form is not parsed and considered Junk? I'm using compare-locales (2.1) as parser, and python-fluent (0.4.3)

The code is as simple as
```
entities, map = file_parser.parse()
for entity in entities:
    print entity, type(entity)
```

Here's what I get
```
default-content-process-count <class 'compare_locales.parser.FluentEntity'>
_junk_1_139-250 <class 'compare_locales.parser.Junk'>
sample <class 'compare_locales.parser.FluentEntity'>
_junk_2_290-323 <class 'compare_locales.parser.Junk'>
```

Reply:
Oh wow, it's the indenting.

```
timeDiffHoursAgo = { $number ->
         [one] 1 hour ago
        *[other] { $number } hours ago
    }
```

I feel more and more that we're building a format usable only by tools :-\

Reply:
Yes, it's the indent. The currently-recommended way of writing this is:

```properties
timeDiffHoursAgo =
    { $number ->
         [one] 1 hour ago
        *[other] { $number } hours ago
    }
```
In https://github.com/projectfluent/fluent/issues/42 we're talking about relaxing the indentation requirement for messages which consist entirely of a single placeable. Zibi and I have a proposal about how to fix this, I'll comment in that issue on Monday.

Reply:
In cases like this it may be helpful to see what Playground says about the message's syntax. Open the `Console` panel to see the error message: http://projectfluent.io/play/?gist=aa08a83ab68ddb34f01ba786836c9e4b

Reply:
Thanks. I'll wait for your comment in the issue. I really think we need to find a compromise between clean and easy to parse, and reality. I copied and pasted that fragment from another file, I didn't notice I changed the indenting, and spend a good 15 minutes trying to figure out where my code was broken.